src/Microsoft.VisualStudio.Extensibility.Testing.Xunit.Shared/Harness/IdeTestAssemblyRunner.cs

```diff
@@ -261,14 +261,14 @@ private static Version GetVersion(VisualStudioVersion visualStudioVersion)
             }
         }
 
-        private VisualStudioVersion GetVisualStudioVersionForTestCase(IXunitTestCase testCase)
+        private Tuple<VisualStudioVersion, string> GetVisualStudioVersionForTestCase(IXunitTestCase testCase)
```

Comment:
Named Tuple would be better for readability.

Reply:
Agreed, but named tuples aren't available in the target frameworks for Visual Studio 2012. :(